apps/server/src/shared/domain/domainobject/tool/tool-reference.ts

```diff
@@ -0,0 +1,21 @@
+import { ToolConfigurationStatus } from './tool-configuration-status';
+
+export class ToolReference {
```

Comment:
Only nice to know. Tools is a seperated topic. Is it easy possible in a additional ticket to move the Domain Object it into the module it self?

Reply:
Yes it should be easy because no other module uses it at the moment. 
I was about to do it but the arc decision is still missing

Reply:
Yes not as part of this PR.